packages/polymath-issuer/src/actions/token.js

```diff
@@ -14,12 +14,14 @@ import type {
   Address,
 } from '@polymathnetwork/js/types';
 
-import { formName as completeFormName } from './components/CompleteTokenForm';
-import { fetch as fetchSTO } from '../sto/actions';
-import { PERMANENT_LOCKUP_TS } from '../compliance/actions';
-import CreatedEmail from './components/CreatedEmail';
-import type { GetState } from '../../redux/reducer';
-import type { ExtractReturn } from '../../redux/helpers';
+// TODO: Here we are creating components inside a Redux action file...
```

Comment:
Two things about this:
- How about we agree on always signing TODO/FIXME comments: `// TODO @RafaelVidaurre: foo` just so that we can always know who to ask for more details
- When doing this it would be even better if we also add what needs to be done (given that it is a `TODO` comment). The other option I guess is to use a `FIXME` which makes it more clear that what we are doing is wrong